syntax = "proto3";

message VesselItem {
    int32 id = 1;
    int32 max_weight = 2;
}

message GetVesselRequest {
    int32 weight = 1;
}

service Vessel {
	rpc GetVessel(GetVesselRequest) returns (VesselItem) {}
}
